src/Workspaces/Core/Portable/Shared/Extensions/SymbolInfoExtensions.cs

```diff
@@ -31,7 +31,9 @@ private static IEnumerable<ISymbol> GetAllSymbolsWorker(this SymbolInfo info)
 
         public static ISymbol GetAnySymbol(this SymbolInfo info)
         {
-            return info.GetAllSymbols().FirstOrDefault();
+            return info.Symbol != null
```

Comment:
Comment as to why? And are there enough other callers to GetAllSymbols that should be looked at too?